ush/config_defaults.sh

```diff
@@ -2230,8 +2230,8 @@ ISEED_SPP=( "4" "5" "6" "7" "8" )
 # at the moment, and is used for each pattern.
 #
 DO_LSM_SPP="FALSE" #If true, sets lndp_type=2
-LSM_SPP_TSCALE=( "21600" "21600" "21600" "21600" "21600" "21600" "21600" )
-LSM_SPP_LSCALE=( "150000" "150000" "150000" "150000" "150000" "150000" "150000" )
+LSM_SPP_TSCALE=( "21600" "21600" "21600" "21600" "21600" )
+LSM_SPP_LSCALE=( "150000" "150000" "150000" "150000" "150000" )
```

Comment:
Where is this change from? The variable list still has 7.

Reply:
Code only can read 5. 7 will crash the model.